azurerm/resource_arm_key_vault.go

```diff
@@ -181,6 +182,18 @@ func resourceArmKeyVault() *schema.Resource {
 				Optional: true,
 			},
 
+			"enabled_for_soft_delete": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				ValidateFunc: validate.BoolIsTrue(),
+			},
+
+			"enabled_for_purge_protection": {
+				Type:         schema.TypeBool,
+				Optional:     true,
+				ValidateFunc: validate.BoolIsTrue(),
```

Comment:
given the code below ignores this if it's not set to `true` - I think we can safely remove this validation function?

Reply:
given the comment in the docs `Once enabled you can not disable this setting anymore!` can we add `ForceNew: true` here?